itemloaders/__init__.py

```diff
@@ -6,12 +6,9 @@
 from collections import defaultdict
 from contextlib import suppress
 
-from scrapy.item import Item
-from scrapy.loader.common import wrap_loader_context
-from scrapy.loader.processors import Identity
-from scrapy.selector import Selector
```

Comment:
I think most css/xpath code can be brought back by importing Selector from parsel instead of scrapy.selector

Reply:
I thought we'd leave `parsel` out for now.
But it makes sense to include ir